metadata-io/src/main/java/com/linkedin/metadata/timeline/eventgenerator/EntityChangeEventGeneratorFactory.java

```diff
@@ -0,0 +1,23 @@
+package com.linkedin.metadata.timeline.eventgenerator;
+
+import com.linkedin.metadata.timeline.data.ChangeCategory;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * A cheap factory for generating EntityChangeEvents, keyed by entity-type, element-type, aspect-name
+ */
+public class EntityChangeEventGeneratorFactory {
+
+  private final Map<String, EntityChangeEventGenerator> _entityChangeEventGeneratorMap = new HashMap<>();
+
+  public void addDiffer(String entityName, ChangeCategory elementName, String aspectName,
```

Comment:
minor: I'd refactor to addGenerator and getGenerator.

Reply:
Done